packages/core/src/conversation/ConversationService/ConversationService.ts

```diff
@@ -288,7 +288,7 @@ export class ConversationService {
     const sentAt = response.time;
 
     const failedToSend =
-      response.failed || Object.keys(response.failed_to_send ?? {}).length > 0
```

Comment:
is response.failed an array or a boolean?

Reply:
array but here we only care if it is there or not, backend does not send it when no failure is there.

Reply:
bit confusing to me when we include it on 281 before overwriting it from the BE

Reply:
right, changed it to undefined
https://github.com/wireapp/wire-web-packages/pull/5187/commits/4c783e3b00096d86e9a6a262a6abed9c484362bb

Reply:
https://github.com/wireapp/wire-web-packages/pull/5187/files#diff-4c1562146ae07c9ddbe8479827807af4a87be8cc13ae150557ad72cf6065f254